### PR TITLE
cleanup code, prepend 'UI-Test' to all selenium messages

### PR DESCRIPTION
**Pull request by dothebart**

### CODE REVIEW

**Comment by dothebart** [comment]
https://jenkins.arangodb.biz/view/Upgrade/job/Upgrade-3.8-to-devel/102/

File: release_tester/arangodb/starter/deployments/selenium_deployments/__init__.py [hunk 1]
```diff
@@ -38,7 +41,7 @@ def init(runner_type: RunnerType,
             print('S: retrying to launch browser')
             time.sleep(2)
     if selenium_worker.lower() == 'chrome':
-        original_size = driver.get_window_size()
+        # original_size = driver.get_window_size()
```
Comment (mchacki) [inline release_tester/arangodb/starter/deployments/selenium_deployments/__init__.py:44]:
```suggestion
```

File: release_tester/arangodb/starter/deployments/selenium_deployments/sbase.py [hunk 2]
```diff
@@ -237,9 +238,7 @@ def detect_version(self):
                     ret['version'], ret['enterprise']))
                 if (len(ret['version']) > 0) and (len(ret['enterprise']) > 0):
                     return ret
-                else:
-                    self.progress('retry version.')
-                    continue
+                self.progress('retry version.')
```
Comment (mchacki) [inline release_tester/arangodb/starter/deployments/selenium_deployments/sbase.py:241]: Was removal of continue intentional here?

File: release_tester/arangodb/starter/deployments/selenium_deployments/__init__.py [hunk 3]
```diff
@@ -61,7 +64,7 @@ def init(runner_type: RunnerType,
 
     if runner_type == RunnerType.DC2DCENDURANCE:
         from arangodb.starter.deployments.selenium_deployments.dc2dc_endurance import Dc2DcEndurance
-        return Dc2DcEndurance(*args)
+        return Dc2DcEndurance(driver)
```
Comment (mchacki) [inline release_tester/arangodb/starter/deployments/selenium_deployments/__init__.py:67]: This seems unrelated. I cannot comment if this is correct change or not, as I am not familiar with the code.

The new version looks more consistent with other functionality, but I do not see counter part modification in Dc2DcEndurance.

**Review by mchacki — APPROVED** [review]
The UI-Test prepend looks good :+1:

There are some inline comments where I do not know details, they may be correct, if they are please close them.

File: release_tester/arangodb/starter/deployments/selenium_deployments/__init__.py (hunk 3 above)
Comment (dothebart) [reply release_tester/arangodb/starter/deployments/selenium_deployments/__init__.py:67]: yes, this was an oversight pointed out by pylint. 

The endurance module isn't frequently used, hence it didn't suface yet.

File: release_tester/arangodb/starter/deployments/selenium_deployments/sbase.py (hunk 2 above)
Comment (dothebart) [reply release_tester/arangodb/starter/deployments/selenium_deployments/sbase.py:241]: pylint pointed out that this was useless - and indeed it was.

**Comment by dothebart** [comment]
one test blue, one dc2dc KI. Merging.
